### PR TITLE
[DOC-7128] Saved views

### PR DESCRIPTION
**Pull request by urbiz-nr**
- Added saved views in Explorer doc
- Updated a couple of screenshots

See DOC-7128 for more info.

### CODE REVIEW

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx [hunk 1]
```diff
@@ -66,23 +66,24 @@ The New Relic Explorer brings together data reported by any [entity](/docs/new-r
   You can create new entity types to monitor any data source. Learn more about [entity synthesis](https://github.com/newrelic/entity-definitions#entity-definitions).
 </Callout>
 
-To use the New Relic Explorer: Go to **[one.newrelic.com](http://one.newrelic.com)** and select **Explorer**.
+Use the New Relic Explorer at **[one.newrelic.com](http://one.newrelic.com)**!
 
-![nrexplorer2.png](./images/nrexplorer2.png "nrexplorer2.png")
+![nrexplorer2.png](./images/nrexplorer20210722.png "nrexplorer2.png")
```
Comment (barbnewrelic) [inline src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx:71]: style guide recommends a good description, not just the filename. see the example in https://docs.newrelic.com/docs/style-guide/processes-procedures/embed-images/#embed

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx [hunk 2]
```diff
@@ -66,23 +66,24 @@ The New Relic Explorer brings together data reported by any [entity](/docs/new-r
...
-3. [New Relic Lookout](#view-lookout). Spot entities that have recent performance deviations. Use this to quickly identify unusual behaviors. Other components include:
-4. [Filterbar](#filter-tag). Drill down and locate problematic entities easily with our improved search capabilities, and benefit from the `AND` and OR operators in place.
-5. See everything. All the different entity types you have access to are listed in the collapsible sidebar on the left of the screen. This sidebar is interactive and used for exploratory purposes, and it allows you to see only entities of the selected type, updating the filter from the filterbar.
-6. Create a new [workload](/docs/new-relic-one/use-new-relic-one/workloads/workloads-isolate-resolve-incidents-faster) for meaningful groupings of your monitored entities.
-7. Add more data to instrument more elements of your system and achieve full stack observability.
+1. **[List](#view-list)**. Browse and filter from a list of all the entities in your account. Use this landing interface to navigate, group and filter your entities.
```
Comment (barbnewrelic) [inline src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx:79]: style guide uses Oxford comma so s/be "navigate, group, and filter"

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx [hunk 3]
```diff
@@ -66,23 +66,24 @@ The New Relic Explorer brings together data reported by any [entity](/docs/new-r
...
+2. **[New Relic Navigator](#view-navigator)**. Get a high density overview of all your entities, grouped by entity type or by tags. Use this to detect any issues and health patterns at a glance.
+3. **[New Relic Lookout](#view-lookout)**. Spot entities that have recent performance deviations. Use this to quickly identify unusual behaviors. 
+4. **Saved views**. Save your favorite filters as a view and recover them in a click each time you return to the Explorer. 
+5. **[Filterbar](#filter-tag)**. Drill down and locate problematic entities easily with our improved search capabilities, and benefit from the `AND` and OR operators in place.
```
Comment (barbnewrelic) [inline src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx:83]: s/be `OR` with the backticks

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx [hunk 4]
```diff
@@ -66,23 +66,24 @@ The New Relic Explorer brings together data reported by any [entity](/docs/new-r
...
+6. **See everything**. All the different entity types you have access to are listed in the collapsible sidebar on the left of the screen. This sidebar is interactive and used for exploratory purposes, and it allows you to see only entities of the selected type, updating the filter from the filterbar.
```
Comment (barbnewrelic) [inline src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx:84]: I'm wondering if this long sentence could be broken like this? "...for exploratory purposes. It also allows you to see only the entities matching the `type` selected. You can also update the filter from the filter bar." I'm not sure I reworded this to keep what you intended to convey, sorry!

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx [hunk 5]
```diff
@@ -66,23 +66,24 @@ The New Relic Explorer brings together data reported by any [entity](/docs/new-r
...
+7. Create a new **[workload](/docs/new-relic-one/use-new-relic-one/workloads/workloads-isolate-resolve-incidents-faster)** for meaningful groupings of your monitored entities.
+8. **Add more data** to instrument more elements of your system and achieve full stack observability.
```
Comment (barbnewrelic) [inline src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx:87]: This new section is excellent! Really helpful info, congrats!

File: src/content/docs/new-relic-one/use-new-relic-one/get-started/introduction-new-relic-one.mdx [hunk 6]
```diff
@@ -21,7 +21,7 @@ redirects:
   To use New Relic One and the rest of our [observability platform](https://one.newrelic.com), join the New Relic family! [Sign up](https://newrelic.com/signup) to create your free account in only a few seconds. Then ingest up to 100GB of data for free each month. Forever.
 </Callout>
 
-![Explorer in New Relic One](./images/NR1-explorer-intro.png "NR1-explorer-intro.png")
+![Explorer in New Relic One](./images/nrone20210722.png "NR1-explorer-intro.png")
```
Comment (barbnewrelic) [inline src/content/docs/new-relic-one/use-new-relic-one/get-started/introduction-new-relic-one.mdx:24]: style guide recommends a good description, not just the filename. see the example in https://docs.newrelic.com/docs/style-guide/processes-procedures/embed-images/#embed

**Review by barbnewrelic — CHANGES_REQUESTED** [review]
I added a few questions and comments inline. After you have a chance to look them over, feel free to use your admin powers to merge directly. No need for a second review request from me. Really nice, congrats!

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx (hunk 1 above)
Comment (urbiz-nr) [reply src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx:71]: Done! I always forget this!

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx (hunk 4 above)
Comment (urbiz-nr) [reply src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx:84]: I've made a couple of tweaks.

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx (hunk 5 above)
Comment (urbiz-nr) [reply src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx:87]: Thanks!

**Review by x8a — APPROVED** [review]
👏🏻

**Pull request merged**